README.md

```diff
@@ -6,8 +6,8 @@ Referencing #2 issue - Free hosting and HTTPS/SSL
 Use danielcolceag.com and digitalocean.com for the hosting.
 
 
-Referencing #4 issue - High Level Architecture
```

Comment:
it was intentionally put that way

Reply:
My bad, I thought you meant to add an additional line break

Reply:
it had to make the architecture ascii art to be seen correctly in a markdown display